### PR TITLE
feat(dashboards): Allow project column in Issue Widgets

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Allows selection of project column for Issue Widgets
Refactors first and last seen to use default date renderers instead

image

### CODE REVIEW

File: static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx [hunk 1]
```diff
@@ -124,8 +124,17 @@ class IssueWidgetQueries extends React.Component<Props, State> {
...
+        id,
+        shortId,
+        title,
+        lifetime,
+        filtered,
+        count,
+        userCount,
+        project,
+        ...resultProps
+      } = group;
```
Comment (priscilawebdev) [inline static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx:137]:
```suggestion
```

File: static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx [hunk 2]
```diff
@@ -124,8 +124,17 @@ class IssueWidgetQueries extends React.Component<Props, State> {
     GroupStore.add(tableResults);
     const transformedTableResults: TableDataRow[] = [];
     tableResults.forEach(group => {
```
Comment (priscilawebdev) [inline static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx:126]:
```suggestion
    tableResults.forEach({
        id,
        shortId,
        title,
        lifetime,
        filtered,
        count,
        userCount,
        project,
        ...resultProps
      }) => {
```

**Comment by priscilawebdev** [comment]
since we are working on these files,  can we replace `group.project.id` with `project.id`?

https://github.com/getsentry/sentry/blob/a1c2243b2660b3843f3f26d414afe668c94f287a/static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx#L187

File: static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx [hunk 3]
```diff
@@ -141,6 +150,7 @@ class IssueWidgetQueries extends React.Component<Props, State> {
         'issue.id': id,
         issue: shortId,
         title,
+        project: project?.slug,
```
Comment (priscilawebdev) [inline static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx:153]: can project be `undefined/null `here? if yes, should we update the types?

Comment (edwardgou-sentry) [reply static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx]: project should be defined in Group as far as I can tell. I haven't seen a case where it hasn't been defined yet. I've updated this to just `project.slug` now. Thanks!

**Review by priscilawebdev — APPROVED** [review]
code lgtm

**Pull request merged**